puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/hammer-cli-katello-master-release.yaml

```diff
@@ -1,15 +0,0 @@
-- job:
-    name: hammer-cli-katello-master-release
-    project-type: pipeline
-    concurrent: false
-    sandbox: false
-    concurrent: false
-    triggers:
-      - github
-    dsl:
-      !include-raw:
-        - pipelines/vars/hammer-cli-katello-master-release.groovy
-        - pipelines/release/rubygemRPMNightlyRelease.groovy
```

Comment:
Depending on what's merged first (https://github.com/theforeman/foreman-infra/pull/936 or this) you can remove this file as well.

Reply:
Good eye, removed!